.changeset/large-melons-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+Add locale `fr-CA` and translate text in `DatePicker`, `FilterDatePicker`, `FilterDateRangePicker`.
```

Comment:
Could we also callout the FilterBar updates here?

Reply:
Oh yes missed it! Thanks!